pom.xml

```diff
@@ -1577,6 +1581,7 @@
         <hbase.version>${hbase-2.3.runtime.version}</hbase.version>
         <hbase.thirdparty.version>3.3.0</hbase.thirdparty.version>
         <zookeeper.version>3.5.7</zookeeper.version>
+        <hadoop.version>3.1.3</hadoop.version>
```

Comment:
Same here https://github.com/apache/hbase/blob/rel/2.3.3/pom.xml#L1341

Reply:
3.1.2 is not the latest version it supports, it's just the default. (Even HBase branch-2.4 and master uses Hadoop 3.1.2 as default)
There seems to be no problem with using Hadoop 3.1.3 with Hbase 2.2 and 2.3, nor with using 3.1.4 with 2.4.

On one hand, we get bug, security and performance fixes when using a later (compatible) version.

On the other hand, simply sticking to the Hbase default version also has merit, as we can avoid the kind of problems that the last upgrade has created.

This only affects the defaults, and the binary distributions, end users are free to build with wathever version they choose.

I am running a set of builds with HBase 2.1/2.2/2.3 and Hadoop 3.0.3/3.1.3/3.1.3 respectively, so we'll know more in about an hour.

I do not have a firm opinon either way, if you think that we should simnply revert PHOENIX-6326, that is also possible.

Provided that the tests with the above combinations are clean, would you prefer the 3.0.3/3.1.3/3.1.3/3.1.4 version or the old 3.0.3/3.1.2/3.1.2/3.1.2 ?

Reply:
I see, I thought we want to keep the same default Hadoop version for HBase profiles as the default Hadoop version kept by the respective HBase versions.

> This only affects the defaults, and the binary distributions, end users are free to build with wathever version they choose.

That's right, end users are free to build with versions of their choice.

> I am running a set of builds with HBase 2.1/2.2/2.3 and Hadoop 3.0.3/3.1.3/3.1.3 respectively, so we'll know more in about an hour.

Great!

Reply:
> I do not have a firm opinon either way, if you think that we should simnply revert PHOENIX-6326, that is also possible.

I also don't have firm opinion either. I think this might take a while doing testing and all, maybe bake in multibranch build for few days and once we see things clear, maybe we should proceed. Given all the work you have done, i think good to proceed with updated Hadoop versions. Only if it turns out to significant tests failing due to Hadoop upgrade, maybe we can revert and take some time with individual HBase profiles. If we go in this situation, please let me know, i will try to test specific HBase profile and try to help ensure the least latest Hadoop version that we can support.